### PR TITLE
feat: add "Is Webiny Free?" FAQ article

### LINKED ISSUE #120: FAQ - is webiny free?

### PR DESCRIPTION
**Pull request by Fsalker**
resolves #120

### CODE REVIEW

File: docs/faq/is-webiny-free.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+id: is-webiny-free
+title: Is Webiny free?
+sidebar_label: Is Webiny free?
+---
+
+Yes, it is! 
+
+Depending on what you use Webiny for, your hosting prices will vary though. If you host a site that rarely changes, you will be hitting the CDN all the time, which is results in negligible expenses.
```
Comment (Pavel910) [inline docs/faq/is-webiny-free.md:9]: `which is results` at the end of the sentence

File: docs/faq/is-webiny-free.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+[Our API](/docs/deep-dive/architecture/api) heavily employs AWS Lambda Functions. It might be worth taking a look at [AWS Labmda Pricing](https://aws.amazon.com/lambda/pricing/): most of the regions cost less than 0.3$ per 1 million requests. Do the math and you will see how cheap that is. 😉
```
Comment (Pavel910) [inline docs/faq/is-webiny-free.md:11]: `AWS Labmda` typo

File: docs/faq/is-webiny-free.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+But there are plenty of free services out there:
+- [MongoDB](https://www.mongodb.com/cloud/atlas) Atlas allows you to host a free MongoDB instance.
+- [AWS](https://aws.amazon.com/) allows you to initially deploy services for free. As developers, we do not have to worry about it at all.
```
Comment (Pavel910) [inline docs/faq/is-webiny-free.md:15]: `As developers, we do not have to worry about it at all.` this is not very clear. Why shouldn't developers worry about it? 

Maybe something like:
`For development purposes, and even small production apps, free tier will mostly likely be enough to cover the usage of resources employed by Webiny.`

File: docs/faq/is-webiny-free.md (hunk 1 above)
Comment (Fsalker) [reply docs/faq/is-webiny-free.md:9]: didn't notice that. fixed

File: docs/faq/is-webiny-free.md (hunk 2 above)
Comment (Fsalker) [reply docs/faq/is-webiny-free.md:11]: fixed

File: docs/faq/is-webiny-free.md (hunk 3 above)
Comment (Fsalker) [reply docs/faq/is-webiny-free.md:15]: sure

fixed

**Comment by Pavel910** [comment]
@Fsalker please resolve the conflict (I think it's `sidebars.json`) and we can merge this 👍

**Pull request merged**